client/src/services/participant.js

```diff
@@ -44,12 +45,21 @@ export const fetchParticipant = async ({ id }) => {
     const { participant } = await resp.json();
     const cohort = await fetchParticipantCohort({ id });
     const postHireStatus = await fetchParticipantPostHireStatus({ id });
+    const status = participant.latestStatuses.length
+      ? participant.latestStatuses[0].status
+      : 'available';
+    const site =
+      status === 'hired'
+        ? await (await fetchSite(participant.latestStatuses[0].siteId)).json()
+        : null;
```

Comment:
Is 'available' a new status? (I was looking at the server const ParticipantStatus for reference)

Reply:
It's what the table labels participants with no status, so I tried to keep the same convention going.